### PR TITLE
Support virtual environment for ansible 2.0.0.2

### PR DESCRIPTION
**Pull request by xumia**
**- What I did**
Install virtual environment for ansible 2.0.0.2

**- How I did it**

**- How to verify it**
. env-201811/bin/activate
The ansible 2.0.0.2 will be used.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by lguohan** [comment]
how to use?  example?

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -147,4 +148,8 @@ RUN azure-cli_bundle_*/installer
 # TODO: if azure-cli contains newer version azure-keyvault, remove this
 RUN ~/lib/azure-cli/bin/python -m pip install azure-keyvault==0.3.7 -U
 
+# Install Virtual Environment
+RUN python -m virtualenv --system-site-packages env-201803
+RUN env-201803/bin/pip install ansible==2.0.0.2
```
Comment (lguohan) [inline dockers/docker-sonic-mgmt/Dockerfile.j2]: 201811?

Comment (lguohan) [reply dockers/docker-sonic-mgmt/Dockerfile.j2]: also, do we have 2.0.0.2-1?

Comment (xumia) [reply dockers/docker-sonic-mgmt/Dockerfile.j2]: Changed to 201811. I only find 2.0.0.2, there are no 2.0.0.2-1.

Comment (xumia) [reply dockers/docker-sonic-mgmt/Dockerfile.j2]: Call the command ". env-201811/bin/activate" to initialize the environment. You will use ansible 2.0.0.2. The other python packages will use the same as system default ones. Only overwrite the ansible and its dependency packages.

**Pull request merged**